### DISCUSSION TITLE
Feedback: Plug for Wasm :-)

### DISCUSSION THREAD

**Discussion by SugarRayLua** [lyte2d/lyte2d#5] [Talk]
@morew4rd, you mentioned on your twitter feed that there were more Wasm downloads then Windows probably due to increased security. I'd like to continue to make a plug for Wasm also being a better cross-platform solution: for instance, I can run lyte2d on my iPad easily by hosting my own local server on my iPad but otherwise wouldn't be able to use lyte2d :-). Keep up the great work!

(Sorry couldn't respond directly on twitter, I discontinued my twitter subscription a little while ago).

**Comment by morew4rd** [comment]
Hey @SugarRayLua! Thanks for your thoughts on this. I plan to keep wasm going as I think it's a very important target. 

This is the main reason why I have started the project with PUC lua and haven't added luajit support yet.  Giving up wasm after luajit would have been just too tempting 😄 

Unfortunately, a lot of the wasm implementations across browsers have a lot to be desired. WebGL2 quality and supported surface is a bit crappy. Doing audio is hard with the current model. List goes on. Perhaps WebGPU will solve some the problems with the graphics. 

Anyway, that's a whole another discussion. :)

**Comment by SugarRayLua** [comment]
Thank you for your continued efforts! 👍😊

**Comment by morew4rd** [comment]
(Just noticed a cool github feature: you can convert issues to discussions. So did it for this one. Nice job github!)
